module_report/check_apis.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+import subprocess
```

Comment:
```suggestion
import subprocess
import sys
```

Oops, I think I mixed this one up, sorry